lib/rules/declaration-block-no-shorthand-property-overrides/index.mjs

```diff
@@ -32,19 +36,21 @@ const rule = (primary) => {
 				const prop = decl.prop;
 				const unprefixedProp = vendor.unprefixed(prop).toLowerCase();
 				const prefix = vendor.prefix(prop).toLowerCase();
-
-				const overrideables = /** @type {Map<string, Set<string>>} */ (
+				const subProperties = /** @type {Map<string, Set<string>>} */ (
 					longhandSubPropertiesOfShorthandProperties
 				).get(unprefixedProp);
+				const resettables = /** @type {Map<string, Set<string>>} */ (
+					shorthandToResetToInitialProperty
+				).get(unprefixedProp);
+				// see microsoft/TypeScript#57228
+				const union = new Set([...(subProperties ?? []), ...(resettables ?? [])]);
```

Comment:
`union` from https://github.com/tc39/proposal-set-methods cannot be used yet

Reply:
[suggest] Let's use `utils/uniteSets` instead:

```suggestion
				const union = uniteSets(subProperties ?? [], resettables ?? []);
```